membership-attribute-service/app/controllers/AttributeController.scala

```diff
@@ -89,6 +89,12 @@ class AttributeController(
       .map(maybeAttributes => ("supporter-product-data", maybeAttributes.getOrElse(None)))
   }
 
+  private def isLiveApp(ua: String): Boolean = ua.matches("""^Guardian(News)?\/.*""")
+  private def upgradeRecurringContributorsOnApps(userAgent: Option[String], attributes: Attributes): Attributes =
+    if (attributes.isRecurringContributor && userAgent.exists(isLiveApp)) {
+      attributes.copy(SupporterPlusExpiryDate = Some(LocalDate.now().plusDays(1)))
```

Comment:
Note - the apps don't yet use `supporterPlus`, so if we want this out quickly then change it to `digitalPack`